src/components/package-list-item/package-list-item.js

```diff
@@ -32,9 +32,12 @@ export default function PackageListItem({
         'is-selected': active
       })}
     >
-      <h5 data-test-eholdings-package-list-item-name>
+      <div
```

Comment:
If not an `h5` is this better semantically as a paragraph tag?

Reply:
I took this commit back out. The headings are more semantically correct, but we need them to be an h3 in one instance and an h4 in another. I'll revisit that later.